.htaccess

```diff
@@ -1,8 +1,8 @@
 RewriteEngine On
 
-# If requested file is in the static folder, serve it
-RewriteCond %{DOCUMENT_ROOT}/static/%{REQUEST_URI} -f
-RewriteRule ^(.*)$ static/$1 [L]
+# If requested file is in the public/static folder, serve it
+RewriteCond %{DOCUMENT_ROOT}/public/static/%{REQUEST_URI} -f
+RewriteRule ^(.*)$ public/static/$1 [L]
```

Comment:
Since this is now dealing with `public/`, we should move this to the top of `public/.htaccess` and remove the bit about the docroot from the readme.

Reply:
I tried, though I'm not able to get this to work, so will leave as is

The issue is that when it's "redirected" from the root .htaccess to public/.htaccess `%{REQUEST_URI}` will change from something like `/5/index.html` to `/public/5/index.html` and we can no longer do the `-f` (file exists) check as the file in is in `/public/static/5/index.html`

Reply:
Oh weird, okay. Please add a comment explaining why we can't move it, so we don't look at this in the future and wonder if it's in the wrong place.